engine/src/main/java/io/camunda/zeebe/engine/state/migration/DbMigrationState.java

```diff
@@ -265,6 +268,14 @@ public DbMigrationState(
 
     signalSubscriptionMigrationState =
         new DbSignalSubscriptionMigrationState(zeebeDb, transactionContext);
+
+    migratedByVersionKey.wrapString("migrated-by-version");
```

Comment:
🔧 Make it a const

Reply:
You mean static? That would be a bug, `DbValue` must not be static because it mutates internal state during serialization.

Reply:
To me it looks like always the same string no?

Reply:
Oh you mean just the string, not the `DbValue`? Sure that works :+1:

Reply:
But the key object is also always the same or not?